randomwallpaper@iflow.space/sourceAdapter.js

```diff
@@ -86,14 +85,24 @@ var UnsplashAdapter = class extends BaseAdapter {
 			'constraintValue': '',
 		};
 
-		this._settings = new SettingsModule.Settings(RWG_SETTINGS_SCHEMA_UNSPLASH);
+		if (id === null) {
+			id = -1;
+		}
+
+		let path = `/org/gnome/shell/extensions/space-iflow-randomwallpaper/sources/unsplash/${id}/`;
```

Comment:
I can't remove this because the whole multiple sources aspect relies on this path + unique id. It's used to seperate source settings for the same type.